Project.toml

```diff
@@ -8,14 +8,14 @@ FastClosures = "9aa1b823-49e4-5ca5-8b0f-3971ec8bab6a"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 LinearOperators = "5c8ed15e-5a4c-59e4-a42b-c7e8811fb125"
 NLPModels = "a4795742-8479-5a88-8948-cc11e1c8c1a6"
-NLPModelsIpopt = "f4238b75-b362-5c4c-b852-0801c9a21d71"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
-FastClosures = "≥ 0.2.1"
-LinearOperators = "0.5.0"
-NLPModels = "0.6.1, < 0.9.0"
-Requires = "≥ 0.3.0"
+FastClosures = "0.2.1, 0.3.0"
+LinearOperators = "0.7.0, 1"
+NLPModels = "0.10.0, 1"
+Requires = "0.3.0, 0.5"
```

Comment:
There is no range in Pkg, yet. This needs to be "0.3, 0.4, 0.5".

Reply:
Isn't [this](https://julialang.github.io/Pkg.jl/v1/compatibility/#Version-specifier-format-1) a range?

Reply:
No, that's a union of two `caret` specifiers: `1.2` implies `[1.2,2.0)` and `2` implies `[2,3)`, which together form the `[1.2.0,0.3.0)` specified.

Reply:
How are the LinearOperators and FastClosures valid then? I obviously understand nothing that that versioning system, and the docs don't help.

Reply:
FastClosures works for `[0.2.1,0.4.0)`, and LinearOperators works for `[0.7,0.8) U [1.0,2.0)`. I think it lacks examples, especially in situations where it differs from ranged conditions. But from what I understood, there are only `~` and `^` (and inequalities, but ignore those?).  `^` means up to, not including, the next semver, and `~` is a more limited version of it. On pre-1.0 releases, the next semver of `0.x.y` is `0.(x+1)`.